In [1]:
import requests

In [2]:
url = "https://aviationweather.gov/api/data/airport?ids=EDDH"

In [3]:
payload = {}
headers = {}

In [4]:
response = requests.request("GET", url, headers=headers, data=payload)

In [5]:
print(response.text)

AIRPORT INFO for 19836
  Name: HAMBURG AIRPORT
  IATA: HAM
  Type: ARP
  Latitude: 53.6318
  Longitude: 9.9951
  Elevation: 8
  Source: Intl
  State: -
  Country: DE
  Mag Declination: 2E
  Use: P
  Services: S
  Tower: T
  Beacon: B
  Passengers: 11090
  Runways
     Number: 2
     Length: L
     Type: H
     Runway: 15/33 Dimension: 12024x150 Surface: Hi Align: 150
     Runway: 05/23 Dimension: 10663x150 Surface: H Align: 48



### 1. What is the name of the airport? 
HAMBURG AIRPORT


### 2. What is the IATA code of the airport? 
HAM

### 3. What is the geolocation (longitude/latitude) of the airport? 
Longitude: 9.9951

Latitude: 53.6318

### 4. How many runways does the airport have? 
Two (15/33 and 05/23)